addons/web_editor/views/snippets.xml

```diff
@@ -380,6 +380,20 @@
     <div data-js="ImageOptimize"
          data-selector="img">
         <t t-call="web_editor.snippet_options_image_optimization_widgets"/>
+        <we-select string="Shape" data-name="shape">
+            <we-button data-shape="">None</we-button>
+            <we-button data-shape="web_editor/blob_1_solid_rd"  data-select-label="Blob 01">Blob 1</we-button>
+            <we-button data-shape="web_editor/blob_2_solid_str" data-select-label="Blob 02">Blob 2</we-button>
+            <we-button data-shape="web_editor/blob_3_solid_rd" data-select-label="Blob 03">Blob 3</we-button>
+            <we-button data-shape="web_editor/organic_1_line" data-select-label="Organic Line 01">Organic Line</we-button>
+            <we-button data-shape="web_editor/organic_2_pattern_dot" data-select-label="Organic Dot 01">Organic Dot</we-button>
+            <we-button data-shape="web_editor/organic_3_pattern_cross" data-select-label="Organic Cross 01">Organic Cross</we-button>
+            <we-button data-shape="web_editor/organic_4_pattern_caps" data-select-label="Organic Caps 01">Organic Caps</we-button>
+            <we-button data-shape="web_editor/oval_1_solid_rd" data-select-label="Oval 1">Oval 1</we-button>
+            <we-button data-shape="web_editor/oval_2_line" data-select-label="Oval 1">Oval 2</we-button>
+            <we-button data-shape="web_editor/oval_3_pattern_line" data-select-label="Oval 1">Oval 3</we-button>
```

Comment:
You don't need both a textContent and a `select-label` data attribute, we'll probably want a pager selector like we currently have for background-shapes, in which case you'll be able to drop the text-content but will need to specify an image to display. (which will probably be the url of the shape itself)